<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>317</width>
    <height>155</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Binary</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLCDNumber" name="lcdNumber">
     <property name="digitCount">
      <number>2</number>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="bit3">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>Bit 3</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="bit2">
       <property name="text">
        <string>Bit 2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="bit1">
       <property name="text">
        <string>Bit 1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="bit0">
       <property name="text">
        <string>Bit 0</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
